src/components/CampaignOverview.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section class="favor-ranking">
+    <table class="favor-ranking-table tableFixHead">
+      <thead class="sticky">
+        <tr>
+          <th class="favor-ranking-rank">Rank</th>
```

Comment:
`<th>` tags should have "id" or "scope" attributes
https://rules.sonarsource.com/html/RSPEC-1102